io-engine/src/subsys/nvmf/subsystem.rs

```diff
@@ -537,6 +538,17 @@ impl NvmfSubsystem {
 
     /// destroy the subsystem
     pub fn destroy(&self) -> i32 {
+        unsafe {
+            if spdk_nvmf_subsystem_remove_ns(self.0.as_ptr(), 1) != 0 {
```

Comment:
Should we check that we're either stopped or not paused? Otherwise this will trigger an assert from spdk.

Reply:
Probably, I better mark this method "unsafe", and make it pub(super). This method is only called from other method, after stopping the subsystem.